cmd/kubectl-gadget/version.go

```diff
@@ -15,16 +15,30 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 
-	"github.com/spf13/cobra"
-
 	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
 	"github.com/inspektor-gadget/inspektor-gadget/internal/version"
 	grpcruntime "github.com/inspektor-gadget/inspektor-gadget/pkg/runtime/grpc"
+	"github.com/spf13/cobra"
```

Comment:
Please run `make lint` to fix the order of these imports. LGTM otherwise.

Reply:
done